src/components/Survey/Category.js

```diff
@@ -49,14 +62,31 @@ export default React.memo(({ category, next }) => {
         />
       ))}
       <div className="actions">
-        {isRequired ? null : (
-          <button type="button" className="skip" onClick={() => nextQuestion()}>
-            Skip
+        <div>
```

Comment:
The div inside a button is semantically incorrect replace it with a `span`

Reply:
This not the case here. we have a button inside a div